src/css/_variables.sass

```diff
@@ -0,0 +1,5 @@
+$primary-color: #010101;
+$secondary-color: #f90254;
```

Comment:
Has the background-color of the footer been changed? It's not the same as what I've got from the design on [Zeplin] - It's `#ff4800`. (https://app.zeplin.io/project/5aab38cd6c05ebc47a367cf2/screen/5aab39e8d4ea96de2994d9b1)

Reply:
Yes, it's been changed, here's the updated one https://app.zeplin.io/project/5aab38cd6c05ebc47a367cf2/screen/5aab39ec9b3829e51d9e5822